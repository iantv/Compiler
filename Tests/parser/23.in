int a = 1;
char c = 'z';
void main(int b){
	double d = 1 + c;
	b = a;
}