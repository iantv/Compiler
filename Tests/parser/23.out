main: function
	b: parameter
		int
	{
		d: variable
			double
		expression:
				1
			=
				a
		expression:
				z
			=
				c
		expression:
							c
						int
					+
						1
				double
			=
				d
		expression:
				a
			=
				b
	}
returns
	void
a: variable
	int
c: variable
	char
